### ISSUE TITLE
Add logs to rest api

### ISSUE THREAD

**Issue by BasvanH** [openhab/openhab-core#3861]
If we added the logs to Rest API we can integrate them on various places like custom widgets and Habpanel without having to build bash/PHP scripts.

We could add /logs
Request a certain logger or all
Optional time range
Optional row range or last X records
Optional pagination

We do you think of this idea?

**Comment by J-N-K** [comment]
There is #3345 and #3859 which seem to address this issue, is that correct?

**Comment by BasvanH** [comment]
Yes it does, great! Are there any filter options available to avoid pulling to much data?

Is this already available in any snapshot?

**Comment by J-N-K** [comment]
No, both not available. Yes, both allow filtering by the logger name with a regular expression.

**Comment by BasvanH** [comment]
> No, both not available.

Do you know when it will be?

**Comment by BasvanH** [comment]
@J-N-K do you have an estimated date?

**Comment by rkoshak** [comment]
@J-N-K has no insight into how long it will take. Once the PRs are completed and reviewed they will be merged and available in the next snapshot build of OH 4.1 (or 4.2 if it doesn't get done before the end of the year.

It's a group effort involving volunteers. There really is no such things as a deadline for anything.

**Comment by florian-h05** [comment]
I think with active work on a log viewer for main UI and the log websocket in place this issue can be closed @openhab/core-maintainers.

**Comment by Nadahar** [comment]
It seems that #3345 will never be (too few SSE connections is a problem already, so probably not a good idea anyway) and #3859 has been merged for a couple of years now.

Is there any reason why this issue is still open?
